packages/lib/utils/recipient-formatter.ts

```diff
@@ -3,7 +3,7 @@ import { Recipient } from '@documenso/prisma/client';
 export const recipientInitials = (text: string) =>
   text
     .split(' ')
-    .map((name: string) => name.slice(0, 1).toUpperCase())
+    .map((name: string) => name[0].toUpperCase())
```

Comment:
I don't think this safeguards against empty strings, it's probably best to leave it as slice unless there is an issue.

Reply:
That's correct!
<img width="151" alt="image" src="https://github.com/documenso/documenso/assets/13398220/a62cccef-8dd5-46d2-a85c-bdc27a07009e">

Reply:
Oops! Thanks for pointing it out, I hadn't considered the empty string case. I will revert this change as well :)